krux/cli.py

```diff
@@ -274,6 +292,12 @@ def add_logging_args(parser):
         choices=LEVELS.keys(),
         help='Verbosity of logging. (default: %(default)s)'
     )
+    group.add_argument(
+        '--log-file',
+        default=None,
```

Comment:
There is an argparse.FileType that you can use but it opens the file and returns the file object from the args, which won't work with WatchedFileHandler. WatchedFileHandler should raise its own error fine when it gets instantiated.